lightnode/docker/lnode-base.dockerfile

```diff
@@ -0,0 +1,38 @@
+# syntax=docker/dockerfile:1
+
+# The base libraries required to build and run the light node.
+
+FROM debian
```

Comment:
This is gonna pull in the entire distro. Recommend using [distroless](https://github.com/GoogleContainerTools/distroless) see also [smallest-golang-docker-image](https://klotzandrew.com/blog/smallest-golang-docker-image/)

Reply:
Thanks! I will look into this.

Reply:
I've updated this branch. It now builds from `golang:1.21.12-bookworm`, and then copies the resulting go binary onto the `scratch` image. Total size of the resulting docker file with the "hello world" placeholder is 3.4 MB.

Reply:
🥳